src/content/developers/tutorials/hello-world-smart-contract.index.md

```diff
@@ -0,0 +1,362 @@
+// Learn more: https://solidity.readthedocs.io/en/v0.5.10/layout-of-source-files.html#pragma
+pragma solidity ^0.7.0;
+
+// Defines a contract named `HelloWorld`.
+// A contract is a collection of functions and data (its state). Once deployed, a contract resides at a specific address on the Ethereum blockchain. Learn more: https://solidity.readthedocs.io/en/v0.5.10/structure-of-a-contract.html
+contract HelloWorld {
+
+   // Declares a state variable `message` of type `string`.
+   // State variables are variables whose values are permanently stored in contract storage. The keyword `public` makes variables accessible from outside a contract and creates a function that other contracts or clients can call to access the value.
+   string public message;
+
+   // Similar to many class-based object-oriented languages, a constructor is a special function that is only executed upon contract creation.
+   // Constructors are used to initialize the contract's data. Learn more:https://solidity.readthedocs.io/en/v0.5.10/contracts.html#constructors
+   constructor(string memory initMessage) {
+   
+      // Accepts a string argument `initMessage` and sets the value into the contract's `message` storage variable).
+      message = initMessage;
+   }
+
+   // A public function that accepts a string argument and updates the `message` storage variable.
+   function update(string memory newMessage) public {
+      message = newMessage;
+   }
+}
+```
+
+This is a super simple smart contract that stores a message upon creation and can be updated by calling the `update` function.
+
+Step 11: Connect Metamask & Alchemy to your project
+===================================================
+
+We’ve created a Metamask wallet, Alchemy account, and written our smart contract, now it’s time to connect the three.
+
+Every transaction sent from your virtual wallet requires a signature using your unique private key. To provide our program with this permission, we can safely store our private key (and Alchemy API key) in an environment file.
+
+> To learn more about sending transactions, check out [this tutorial](/sending-transactions-using-web3-and-alchemy) on sending transactions using web3.
+
+First, install the dotenv package in your project directory:
+
+```
+npm install dotenv --save
+```
+
+Then, create a `.env` file in the root directory of our project, and add your Metamask private key and HTTP Alchemy API URL to it.
+
+*   Follow [these instructions](https://metamask.zendesk.com/hc/en-us/articles/360015289632-How-to-Export-an-Account-Private-Key) to export your private key
+*   See below to get HTTP Alchemy API URL
+
+![get alchemy api key](https://user-images.githubusercontent.com/23513719/111497267-f682ef00-870e-11eb-92ba-e93abd7dc1a3.gif)
+
+Copy Alchemy API URL
+
+Your `.env` should look like this:
+
+```
+API_URL = "https://eth-ropsten.alchemyapi.io/v2/your-api-key"  
+PRIVATE_KEY = "your-metamask-private-key"
```

Comment:
If this is the approach being taken, I would recommend a bold alert somewhere emphasizing that this should be considered a burner wallet for dev testing only and to never use the address to this private key on mainnet with real ETH... probably a general precaution worth reminding new users of either way.

Reply:
Any specific suggestions for the approach users should take when deploying to mainnet?

Reply:
Not off the top of my head, was thinking just a note about being careful is all...  don't place a private key with real ETH linked to it and then commit your code for example. Never hurts to remind people about general private key security